cmd/desync/list.go

```diff
@@ -17,16 +20,22 @@ func newListCommand(ctx context.Context) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "list-chunks <index>",
 		Short: "List chunk IDs from an index",
-		Long: `Reads the index file and prints the list of chunk IDs in it. Use '-' to read
-the index from STDIN.`,
-		Example: `  desync list-chunks file.caibx`,
-		Args:    cobra.ExactArgs(1),
+		Long: `Reads the index file and prints the list of chunk IDs in it. The '--detailed' option shows more
+information about the chunks, like their uncompressed and compressed size. In order to include the compressed size info,
+you should use the '--store' option as well, to set in which local store the compressed chunks are stored in.
+With the '--detailed' option, the output will be in the JSON format and could be reused in 'desync info --chunks-info'.
+Use '-' to read the index from STDIN.`,
+		Example: `  desync list-chunks file.caibx
+desync list-chunks --store /mnt/store --detailed file.caibx`,
+		Args: cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return runList(ctx, opt, args)
 		},
 		SilenceUsage: true,
 	}
 	flags := cmd.Flags()
+	flags.BoolVarP(&opt.detailed, "detailed", "d", false, "detailed chunk info")
+	flags.StringVarP(&opt.store, "store", "s", "", "local source store")
```

Comment:
Yeah, makes sense. I created `inspect-chunks` because I kept the local store as optional. If you don't provide a local store, the JSON will only contain the list of chunks hashes and uncompressed sizes.

Probably not something that is terribly useful to have, but I tired to keep this new command as generic as possible. In this way, if in the future we want to add other elements as well, it should be easier to expand.

Of course I'm open to suggestions here.